cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/batch/BatchJoinerRuntimeContext.java

```diff
@@ -28,13 +28,6 @@
 public interface BatchJoinerRuntimeContext extends BatchRuntimeContext {
 
   /**
-   * Returns a map of input schemas for joiner. If the input schema for an input is null, the map will have input
-   * name with null schema for that input
-   * @return a map of input stage name to input schema for all the inputs to joiner
-   */
-  Map<String, Schema> getInputSchemas();
```

Comment:
Are we removing this because `getInputSchemas` method is now part of `StageContext`? Should we also remove the `getOutputSchema` then? This interface can then be deprecated?